CPT331.Core/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿#region Using References
+
+using System;
+
+#endregion
+
+namespace CPT331.Core.Extensions
+{
+	public static class StringExtensions
+	{
+		public static bool EqualsIgnoreCase(this string value, string compareTo)
+		{
+			return value.Equals(compareTo, StringComparison.OrdinalIgnoreCase);
```

Comment:
So I believe that OrdinalIgnoreCase is fastest for this kind of comparison where language mechanics don't really matter.